parser/src/cfg/mod.rs

```diff
@@ -1383,13 +1390,14 @@ Params in order:
     }))))
 }
 
-fn parse_tap_hold_release_keys(
+fn parse_tap_hold_keys(
     ac_params: &[SExpr],
     s: &ParsedState,
+    custom_func: TapHoldCustomFunc,
 ) -> Result<&'static KanataAction> {
     if ac_params.len() != 5 {
         bail!(
-            r"tap-hold-release-keys expects 5 items after it, got {}.
+            r"tap-hold-(release|except)-keys expects 5 items after it, got {}.
```

Comment:
IIRC this pattern of `(option1|option2)`exists in other parts of parsing but looking back I don't like it so much (and I'll go change the existing occurrences later... if I remember). Please instead pass in the action name and use it directly in the error message.

Reply:
done